app/main.py

```diff
@@ -120,4 +118,4 @@ def battle(list_of_khights: list[KnightsTemplate]) -> dict:
     }
 
 
-print(battle(khights_types))
+print(battle())
```

Comment:
im not adding any parameters but
FAILED tests/test_main.py::test_base_knights - TypeError: battle() takes 0 positional arguments but 1 was given
FAILED tests/test_main.py::test_lancelot_overpowered - TypeError: battle() takes 0 positional arguments but 1 was given
FAILED tests/test_main.py::test_red_knight_overpowered - TypeError: battle() takes 0 positional arguments but 1 was given
FAILED tests/test_main.py::test_lancelot_has_armour - TypeError: battle() takes 0 positional arguments but 1 was given
FAILED tests/test_main.py::test_mordred_sword_is_not_poisoned - TypeError: battle() takes 0 positional arguments but 1 was given
FAILED tests/test_main.py::test_arthur_armour_weak - TypeError: battle() takes 0 positional arguments but 1 was given
FAILED tests/test_main.py::test_arthur_and_lancelot_have_potion - TypeError: battle() takes 0 positional arguments but 1 was given